src/Server/Game/Game.py

```diff
@@ -109,7 +111,8 @@ def __collect_players(self):
             mess = BaseMessage(sermess.MessageType.FIRST_PLAYER, sermess.Target.SCREEN)
             Server.get_instance().send_message(mess, self.__first_player_id)
         if len(connected_players) == self.__human_player_number:
-            self.__start_game()
+            Timer.sch_fun(15, self.__start_game, ())
+            self.__game_start_signaled = True
```

Comment:
What are you waiting for? Couldn't go on when you receive that? (And not wait a random time and hope it arrives during that.)